### PR TITLE
[OSDEV-903] Add details to doc/release/RELEASE-NOTES.md after changes related to OSDEV-702

### PR DESCRIPTION
**Pull request by mazursasha1990**
[OSDEV-903](https://opensupplyhub.atlassian.net/browse/OSDEV-903) Add records to the doc/release/RELEASE-NOTES.md that describe implemented changes in the issue OSDEV-702

### CODE REVIEW

**Review by killev — APPROVED** [review]

File: doc/release/RELEASE-NOTES.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Architecture/Environment changes
 * [OSDEV-910](https://opensupplyhub.atlassian.net/browse/OSDEV-910) Add separated code quality pipelines for contricleaner, countries, django-api and frontend. After checking, it creates a code coverage report showing each particular app's code coverage. Add separated code quality jobs for code formatters.
+* [OSDEV-702](https://opensupplyhub.atlassian.net/browse/OSDEV-702) Add `contricleaner` as a separate module that parses and validates data passed to him from any source (json/csv/xls). Move `countries` to a separate module so that it becomes possible to use both Django and ContriCleaner.
```
Comment (vladsha-dev) [inline doc/release/RELEASE-NOTES.md:24]: > to him
He is not alive.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
Left one comment

File: doc/release/RELEASE-NOTES.md (hunk 1 above)
Comment (roman-stolar) [reply doc/release/RELEASE-NOTES.md:24]: +1

Comment (mazursasha1990) [reply doc/release/RELEASE-NOTES.md]: Done

**Review by roman-stolar — APPROVED** [review]
LGTM

**Pull request merged**